Automattic/wp-calypso
# Framework: Fix logged-out masterbar logo link

Fixes #5419 

Change the WP logo in the logged-out Calypso masterbar to specifically link to https://wordpress.com, instead of Calypso /, which doesn't work when logged-out (Reader is logged-in only).

<img width="206" alt="screen shot 2016-05-19 at 11 54 39" src="https://cloud.githubusercontent.com/assets/7767559/15391356/9427c89a-1db8-11e6-8790-fd3f991e4c85.png">
Now links to:
<img width="1280" alt="screen shot 2016-05-19 at 11 54 57" src="https://cloud.githubusercontent.com/assets/7767559/15391362/97522664-1db8-11e6-92c0-1db0ac5ddb60.png">

**To Test**
In an incognito window, make sure that the WordPress.com logo in the logged-out master bar at
http://calypso.localhost:3000/start or http://calypso.localhost:3000/design
links to https://wordpress.com
